drivers/src/nrfx_nfct.c

```diff
@@ -556,6 +556,37 @@ nrfx_err_t nrfx_nfct_tx(nrfx_nfct_data_desc_t const * p_tx_data,
     return err;
 }
 
+nrfx_err_t nrfx_nfct_tx_bits(nrfx_nfct_data_desc_t const * p_tx_data,
+                             nrf_nfct_frame_delay_mode_t   delay_mode)
+{
+    NRFX_ASSERT(p_tx_data);
+    NRFX_ASSERT(p_tx_data->p_data);
+
+    if (p_tx_data->data_size == 0)
+    {
+        return NRFX_ERROR_INVALID_LENGTH;
+    }
+
+    // calculate the buffer length in bytes based on the number of bits,
+    // adding an additional byte if there is data beyond the last whole byte.
+    uint32_t buffer_length = NRFX_NFCT_BITS_TO_BYTES(p_tx_data->data_size);
+    if (p_tx_data->data_size & 0b0111)
```

Comment:
Could you use a define for 0b0111 ? Something like #define NRFX_NFCT_BITS_MSK 0x07.
I would also suggest to use hexadecimal numbers

Reply:
Did a fix.